### PR TITLE
Jsf22 resource library contract tests faces40 repeat

### PR DESCRIPTION
**Pull request by pnicolucci**
for #22369

### CODE REVIEW

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
Java 17 build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_MRxYAFOfEe2P_cwgExA7aA

**Comment by pnicolucci** [comment]
Java 17 tests were successful

File: dev/com.ibm.ws.jsf.2.2_fat/test-applications/TestResourceContractsDirectory.war/resources/contracts/manager/template.xhtml [hunk 1]
```diff
@@ -12,28 +12,27 @@
...
-
-	<div id="bottom" class="top">
-		<ui:insert name="bottom">This must be template for manager</ui:insert>
-	</div>
-</h:body>
+        <div id="bottom" class="top">
+            <ui:insert name="bottom">This must be template for manager</ui:insert>
+        </div>
+    </h:body>
 </html>
```
Comment (isaacrivriv) [inline dev/com.ibm.ws.jsf.2.2_fat/test-applications/TestResourceContractsDirectory.war/resources/contracts/manager/template.xhtml]: Optional new line at EoF here

**Review by isaacrivriv — DISMISSED** [review]
Changes look good, optional comment if you want to address it. Thanks!

File: dev/com.ibm.ws.jsf.2.2_fat/test-applications/TestResourceContractsDirectory.war/resources/contracts/manager/template.xhtml (hunk 1 above)
Comment (pnicolucci) [reply dev/com.ibm.ws.jsf.2.2_fat/test-applications/TestResourceContractsDirectory.war/resources/contracts/manager/template.xhtml]: done

**Comment by pnicolucci** [comment]
#libby #build

**Review by isaacrivriv — APPROVED** [review]
Looks good, thanks for making the change!

**Pull request merged**